[2024-03-11T13:57:44.755+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2021-09-17T00:00:00+00:00 [queued]>
[2024-03-11T13:57:44.826+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2021-09-17T00:00:00+00:00 [queued]>
[2024-03-11T13:57:44.830+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-11T13:57:44.923+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): second_task> on 2021-09-17 00:00:00+00:00
[2024-03-11T13:57:44.941+0000] {standard_task_runner.py:60} INFO - Started process 2561 to run task
[2024-03-11T13:57:44.965+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'scheduled__2021-09-17T00:00:00+00:00', '--job-id', '641', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpznpe93az']
[2024-03-11T13:57:44.972+0000] {standard_task_runner.py:88} INFO - Job 641: Subtask second_task
[2024-03-11T13:57:45.219+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v2.second_task scheduled__2021-09-17T00:00:00+00:00 [running]> on host 26e63b1d0d39
[2024-03-11T13:57:45.542+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2021-09-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-09-17T00:00:00+00:00'
[2024-03-11T13:57:45.546+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-11T13:57:45.550+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and I ran after task1']
[2024-03-11T13:57:45.586+0000] {subprocess.py:86} INFO - Output:
[2024-03-11T13:57:45.596+0000] {subprocess.py:93} INFO - hey, I am task2 and I ran after task1
[2024-03-11T13:57:45.600+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-11T13:57:45.680+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20210917T000000, start_date=20240311T135744, end_date=20240311T135745
[2024-03-11T13:57:45.742+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-11T13:57:45.808+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
